plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestingScyllaServer.java

```diff
@@ -52,13 +52,11 @@
     private final CassandraSession session;
 
     public TestingScyllaServer()
-            throws Exception
```

Comment:
This is test code. In tests is often more convenient to just declare `throws Exception` 
and keep it, so that you won't need to add it ever again

Can we exclude the check on test resources?